src/admin-views/editor/fieldset/price.php

```diff
@@ -23,11 +23,20 @@
 
 $is_paypal_ticket = $provider instanceof Tribe__Tickets__Commerce__PayPal__Main || $provider instanceof \TEC\Tickets\Commerce\Module;
 
+// Determine whether or not free tickets are allowed.
+$is_free_ticket_allowed = true;
+if ( $provider instanceof Tribe__Tickets__Commerce__PayPal__Main ) {
+	$is_free_ticket_allowed = false;
+}
+if ( $provider instanceof \TEC\Tickets\Commerce\Module ) {
+	$is_free_ticket_allowed = \TEC\Tickets\Commerce\Settings::is_free_ticket_allowed();
+}
```

Comment:
Let these poor things breathe - spacing!

```suggestion
// Determine whether or not free tickets are allowed.
$is_free_ticket_allowed = true;

if ( $provider instanceof Tribe__Tickets__Commerce__PayPal__Main ) {
	$is_free_ticket_allowed = false;
}

if ( $provider instanceof \TEC\Tickets\Commerce\Module ) {
	$is_free_ticket_allowed = \TEC\Tickets\Commerce\Settings::is_free_ticket_allowed();
}
```

Reply:
You are inconsistent when it comes to this, see `filter_tickets_editor_config()` above.

Put space between control structures.

Reply:
My reasoning (not that it's correct, but maybe you'll understand why): When I comment what action is being taken, I tend to not want to leave a space when it's a multistep process. Int eh above code, I commented `// Determine whether or not free tickets are allowed.`, then wanted to make it clear that the next few steps were achieving just that. I'll change it, though. ;-)